project-base/tests/App/Acceptance/acceptance/PageObject/Front/OrderPage.php

```diff
@@ -112,17 +121,25 @@ public function fillPersonalInfo($firstName, $lastName, $email, $telephone)
      */
     public function fillBillingAddress($street, $city, $postcode)
     {
-        $this->tester->fillFieldByName('order_personal_info_form[street]', $street);
-        $this->tester->fillFieldByName('order_personal_info_form[city]', $city);
-        $this->tester->fillFieldByName('order_personal_info_form[postcode]', $postcode);
+        $streetInput = $this->webDriver->findElement(WebDriverBy::name('order_personal_info_form[street]'));
+        $cityInput = $this->webDriver->findElement(WebDriverBy::name('order_personal_info_form[city]'));
+        $postcodeInput = $this->webDriver->findElement(WebDriverBy::name('order_personal_info_form[postcode]'));
+
+        $this->tester->fillFieldByElement($streetInput, $street);
+        $this->tester->fillFieldByElement($cityInput, $city);
+        $this->tester->fillFieldByElement($postcodeInput, $postcode);
+
+        $this->tester->waitForAjax();
```

Comment:
this waitForAjax should be in every `fill` method on the orderPage, isn't it? (ajax call to save form is performed on each field)

Reply:
Well I wanted to wait after personal and billing info is finished so I am ok with it as it is.